dictionaries/en_AU/src/additional_words.txt

```diff
@@ -1,30 +1,16 @@
-# Add words below
-# Consider adding words to ./shared-additional-words.txt if they are used by all flavours of English.
+# The CSpell English dictionary uses the SCOWL (Spell Checker Oriented Word Lists) dictionary as a
```

Comment:
This isn't true for en_AU or the other non-en-US languages.

Most of the English variant languages are sourced from https://github.com/marcoagpinto/aoo-mozilla-en-dict and other sources.

Getting changes made to the source dictionaries can sometimes be challenging and take a lot of time. `additional_words.txt` is a way to add words in a timely fashion.

Reply:
I updated the code to something simpler,

Please note it mentions the README file that is added by #4397

- #4397

You can review anyway